OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -200,6 +207,10 @@ void SyncMultiplayerCount()
 				{
 					Players.Players.Remove(name);
 					OnPlayerRemoved();
+
+					foreach (var preview in previews)
+						if (preview.Owner == kv.Value)
+							preview.Owner = worldOwner;
```

Comment:
I still disagree with the addition of this line. I think we should avoid doing this until a move tool has been implemented